crates/driver/src/print_build.rs

```diff
@@ -335,6 +335,12 @@ async fn print_file(
                                 t.push(manual_ref.target_value.clone())
                             }
                         },
+                        Directive::AttrStringList(attr) => {
+                            let t = extra_kv_pairs
```

Comment:
nit: why not chain all the way?
```rust
extra_kv_pairs
  .entry(..
  .or_default()
  .push(attr.value.clone())
```

Reply:
Refactored to `append_key_values` function in 525096d2c8e7dd0fc2228985b2c42cdb754d76d9